download_models/downloadOVMSModels.sh

```diff
@@ -183,7 +187,18 @@ else
     echo "Downloading yolov5s models..."
     # Yolov5s INT8
     getModelFiles $yolov5s $pipelineZooModel"yolov5s-416_INT8" $modelPrecisionFP16INT8
-    getProcessFile $yolov5s $pipelineZooModel"yolov5s-416" $yolov5s
+    getProcessFile $yolov5s $pipelineZooModel"yolov5s-416" $yolojson $yolov5s $modelPrecisionFP16INT8
+fi
+
+yolov5ModelFile="${PWD}/$yolov5s/$modelPrecisionFP32INT8/1/$yolov5s.bin"
+echo "$yolov5ModelFile"
+if [ -f "$yolov5ModelFile" ]; then
+    echo "yolov5s model already exists, skip downloading..."
+else
+    echo "Downloading yolov5s models..."
+    # Yolov5s INT8
```

Comment:
- probably want to add FP32 to echo so it's more obvious which one is doing 
- fix the comment